test/libsolidity/semanticTests/array/copying/copy_internal_function_array_to_storage.sol

```diff
@@ -18,8 +18,10 @@ contract C {
 }
 
 // ====
-// compileViaYul: also
 // compileToEwasm: also
+// compileViaYul: also
 // ----
 // one() -> 3
+// gas legacy: 154760
```

Comment:
Why is `gas ir*`  missing here? I would have expected to see `gas ir*` expectations here too, because `compileViaYul` is defined.

Reply:
See https://github.com/ethereum/solidity/pull/10863#discussion_r566840010